plugin.go

```diff
@@ -57,19 +59,26 @@ func (p *Plugin) CobraAddCommands(rootCmd *cobra.Command) error {
 			if err != nil {
 				log.Fatal("error while getting keyringPassphrase option value: ", err)
 			}
+			verbose, err := cmd.Flags().GetCount("verbose")
```

Comment:
Not sure `verbose` is best naming here, maybe `verboseCount` or `verboseDepth` ?